qaframework-bdd-tests/src/test/resources/features/refapp-2.x/02-login/verifyLogin.feature

```diff
@@ -0,0 +1,11 @@
+Feature: User based login
+
+@selenium
+@defaultLogin
+Scenario Outline: User Based login verification
+   When user verifies modules available on home page
+   And user verifies modules available on homepage after login as clerk
+   And user verifies modules available on home page after login as Docter
+   And user verifies modules available on home page after login as Nurse
+   And user verifies modules available on home page after login as sysadmin
+   Then system goes back to login page
```

Comment:
Add an empty line at the EOF to cater for git collaborators

Reply:
Resolved.Thanks.

Reply:
@kdaud  Let me know if this looks fine.

Reply:
@sherrif10 take a second look at the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/342/files#r806469786)

Reply:
@sherrif10 did you see the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/342/files#r809015231)